src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: $card-width;
+  border: $card-border;
```

Comment:
The variable `$card-border` is not defined in this file. Ensure it is defined in a global SCSS file or imported correctly.